lib/opensusebasetest.pm

```diff
@@ -788,7 +788,7 @@ sub wait_boot {
             #assert_screen "dm-password-input", 10;
             elsif (check_var('DESKTOP', 'gnome')) {
                 # In GNOME/gdm, we do not have to enter a username, but we have to select it
-                if (is_tumbleweed) {
+                if (is_tumbleweed || is_sle('>=15-sp2')) {
```

Comment:
please turn around the logic and handle older products as the exception, not the other way around.